test/format/test_format.py

```diff
@@ -456,34 +456,53 @@
     "Sqrt[1/(1+1/(1+1/a))]": {
         "msg": "SqrtBox",
         "text": {
-            "System`StandardForm": "Sqrt[1 / (1+1 / (1+1 / a))]",
-            "System`TraditionalForm": "Sqrt[1 / (1+1 / (1+1 / a))]",
-            "System`InputForm": "Sqrt[1 / (1 + 1 / (1 + 1 / a))]",
-            "System`OutputForm": "Sqrt[1 / (1 + 1 / (1 + 1 / a))]",
+            "System`StandardForm": "1 / Sqrt[1+1 / (1+1 / a)]",
```

Comment:
Why the change in test?

Reply:
This is the effect of implementing the associativity of powers. In master, 

```
Sqrt[1 / (1 + 1 / (1 + 1 / a))]//FullForm
```

produces
```
Power[Power[Plus[1, Power[Plus[1, Power[a, -1]], -1]], -1], Rational[1, 2]]
```

while here,
```
Power[Plus[1, Power[Plus[1, Power[a, -1]], -1]], Rational[-1, 2]]
```

On the other hand, looking at WMA, it seems that it does not implement the associativity in this case either.